element_animal/version.py

```diff
@@ -1,2 +1,2 @@
 """Package metadata."""
-__version__ = '0.1.0b0'
\ No newline at end of file
+__version__ = '0.1.0b1'
```

Comment:
Updating the version will trigger a tag and release.  Should we hold off on releasing until we have finalized the `element-array-ephys` NWB export function?